doc/DOCUMENTATION.org

```diff
@@ -3389,54 +3389,96 @@ Examples:
-| ~SPC k r~   | raise expression (replace parent expression by current one)         |
-| ~SPC k s~   | forward slurp expression                                            |
-| ~SPC k S~   | backward slurp expression                                           |
-| ~SPC k t~   | transpose expression                                                |
-| ~SPC k u~   | undo                                                                |
-| ~SPC k U~   | got to parent sexp backward                                         |
-| ~SPC k C-r~ | redo                                                                |
-| ~SPC k v~   | switch to =visual state=                                            |
-| ~SPC k V~   | switch to =visual line state=                                       |
-| ~SPC k C-v~ | switch to =visual block state=                                      |
-| ~SPC k w~   | wrap expression with parenthesis                                    |
-| ~SPC k W~   | unwrap expression                                                   |
-| ~SPC k y~   | copy expression                                                     |
+| ~SPC k Dw~  | delete word backward                                                |
+| ~SPC k dx~  | delete sexp: ~(as <point> a ..)~ -> ~(as <point> ..)~               |
+| ~SPC k Dx~  | delete sexp backward: ~(as a <point> ..)~ -> ~(as <point> ..)~      |
+| ~SPC k e~   | splice, killing forward: ~(as (bs <point> cs) ds)~ -> ~(as bs ds)~  |
+| ~SPC k E~   | splice, killing backward: ~(as (bs <point> cs) ds)~ -> ~(as cs ds)~ |
+| ~SPC k r~   | raise: ~(as <point> b ..)~ -> ~<point>b~                            |
+| ~SPC k W~   | unwrap sexp: ~(as)~ -> ~as~                                         |
+|-------------+---------------------------------------------------------------------|
+|             | Hybrid (= better for non-Lisp languages) commands                   |
+|-------------+---------------------------------------------------------------------|
+| ~SPC k ` k~ | hybrid delete sexp                                                  |
+|-------------+---------------------------------------------------------------------|
+| Note 1:     | ~xs~ is one or multiple sexp, ~x~ is a single sexp                  |
+| Note 2:     | point is at ~<point>~ when presented, or in the list otherwise      |
+
+****** Bindings from normal state available in Lisp state as well
+
+For convenience, a number of normal state evil binding are available in lisp state as well:
+
+| Key binding | Description                                           |
+|-------------+-------------------------------------------------------|
+| ~SPC k :~   | ex command                                            |
+| ~SPC k i~   | switch to =insert state=                              |
+| ~SPC k I~   | switch to =insert state= at beginning of current line |
+| ~SPC k v~   | switch to =visual state=                              |
```

Comment:
the these three, v, V, C-v:
- They are switch to `visual` state. there's no `visual line` state or `visual block` state
- The description can be: switch to =visual= state and start character-wise/line-wise/block-wise selection.

Reply:
Also, don't quote `state`. just quote `insert` or `visual`.
These symbols are defined in evil and are not arbitrary.

Reply:
You can see the defined state with:
```elisp
(mapcar #'car evil-state-properties)
```

Reply:
Done.